content/source/docs/extend/writing-custom-providers.html.md

```diff
@@ -853,7 +853,7 @@ func flattenContainerSpec(in *server.ContainerSpec) []interface{} {
 func flattenServiceMounts(in []mount.Mount) []map[string]interface{} {
   var out = make([]map[string]interface{}, len(in), len(in))
   for i, v := range in {
-    m := make(map[string interface{}])
+    m := make(map[string] interface{})
```

Comment:
should be `make(map[string]interface{})`

Reply:
Will add a fix before merging! 👍🏼